parameter_files/ProjectParameters.yml

```diff
@@ -0,0 +1,90 @@
+default:
```

Comment:
I'd still love to see documentation of what all these options are, and what their potential values are, and what is the effect of those potential values. I still scan through these and think I'm only confident about what < 50% of these do.

Reply:
👍 Will add that to my todos.